src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -305,7 +305,9 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConfirmBidState
     action_name: Schema.ActionNames.BidFlowPlaceBid,
   })
   bidPlacedSuccessfully(positionId) {
-    queryForBidderPosition(positionId).then(this.checkBidderPosition.bind(this))
+    queryForBidderPosition(positionId)
```

Comment:
perhaps I'm not understanding the code fully, but seems like the [below diff](https://github.com/artsy/emission/pull/1743/files#diff-8a3a841add9adeac68c88de41a452cc4R319) is within the polling code? What's the user flow that is supported by this code path?